### PR TITLE
Updated chrome remote desktop module for slurm image support

### PR DESCRIPTION
**Pull request by saltysoup**
### Submission Checklist

* [ ] Have you installed and run this change against pre-commit? (`pre-commit
  install`)
* [ ] Are all tests passing? (`make tests`)
* [ ] Have you written unit tests to cover this change?
* [ ] Is unit test coverage still above 80%?
* [ ] Have you updated all applicable documentation?
* [ ] Have you followed the guidelines in our Contributing document?

### CODE REVIEW

**Comment by saltysoup** [comment]
Changes made
1. Added block conditional on existing `configure-grid-drivers.yaml` playbook to skip installing drivers if already present ie. Slurm's HPC ubuntu image. The other remaining tasks in the playbook are still run to install xfce desktop, chrome remote desktop and virtualGL drivers.
1. Added blueprint example of how the updated module can be used with slurm

File: community/examples/slurm-chromedesktop.yaml [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    use: [network1]
+    settings:
+      local_mount: /home
+ 
+  - id: remote-desktop
+    source: community/modules/remote-desktop/chrome-remote-desktop
+    use: [network1]
+    settings:
+      install_nvidia_driver: true
+      instance_count: 0
```
Comment (cboneti) [inline community/examples/slurm-chromedesktop.yaml:45]: I would add a comment here to explain that instance count=0 will only create the scripts.

**Review by cboneti — APPROVED** [review]
Just a minor comment, please let me know if you can update the examples list. Otherwise, approved.

**Comment by saltysoup** [comment]
@cboneti - yamllint is reporting syntax errors + wrong indentation for the `remote_desktop_startup` module, with the following error message (although it looks correct). Can you take a look? happy for you to update if required

Getting this error message
```
  51:11     error    wrong indentation: expected 8 but found 10  (indentation)
  54:9      error    syntax error: expected <block end>, but found '<block sequence start>' (syntax)
```

From this module
```
      runners:
      - type: ansible-local
        source: modules/chrome-remote-desktop/scripts/configure-grid-drivers.yml
        destination: /tmp/configure-grid-drivers.yml
      - type: ansible-local
        source: modules/chrome-remote-desktop/scripts/configure-chrome-desktop.yml
        destination: /tmp/configure-chrome-desktop.yml
      - type: ansible-local
        source: modules/chrome-remote-desktop/scripts/disable-sleep.yml
        destination: /tmp/disable-sleep.yml
```

**Review by cboneti — APPROVED** [review]

**Pull request merged**